.github/workflows/documentation.yml

```diff
@@ -20,10 +30,10 @@ jobs:
       with:
         ref: ${{ github.event.pull_request.head.ref }}
 
-    - name: Render terraform docs and push changes back to PR
-      uses: terraform-docs/gh-actions@f6d59f89a280fa0a3febf55ef68f146784b20ba0 # v1.0.0
-      with:
-        working-dir: .
-        output-file: README.md
-        output-method: inject
-        git-push: "true"
+      - name: Render terraform docs and push changes back to PR
```

Comment:
is this indentation correct?

Reply:
added extra indent due to merge queue addition honestly don't know if it's going to work of not